.kitchen_debian.yml

```diff
@@ -1,30 +1,31 @@
 ---
 driver:
   name: vagrant
-
 provisioner:
   name: ansible_playbook
   test_repo_uri: https://github.com/TelekomLabs/tests-os-hardening.git
   hosts: all
   require_ansible_omnibus: true
   ansible_omnibus_url: https://raw.githubusercontent.com/hardening-io/ansible-os-hardening/master/ansible-install.sh
-
 platforms:
 - name: debian-6
   driver_config:
     box: debian-6
-    box_url: https://s3.eu-central-1.amazonaws.com/ffuenf-vagrantboxes/debian/debian-6.0.10-amd64_virtualbox.box
+    box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_debian-6.0.10_chef-provisionerless.box
```

Comment:
There is no reason for http, thanks @ypid for pointing this out. I updated the PR